### ISSUE TITLE
For clippy, --message-format needs to be passed before --

### ISSUE THREAD

**Issue by sebschmi** [intellij-rust/intellij-rust#6834]
## Environment

* **IntelliJ Rust plugin version:** 0.3.140.3644-203
* **Rust toolchain version:** 1.50.0 (cb75ad5db 2021-02-10) x86_64-unknown-linux-gnu
* **IDE name and version:** IntelliJ IDEA 2020.3.2 Community Edition (IC-203.7148.57)
* **Operating system:** Linux 5.4.0-65-generic
* **Macro expansion engine:** new
* **Name resolution engine:** old

## Problem description
My clippy Run configuration fails. It looks like this: `clippy --all --all-targets -- -D warnings`
What gets executed is this: `/home/sebschmi/.cargo/bin/cargo +stable clippy --color=always --all --all-targets -- -D warnings --message-format=json-diagnostic-rendered-ansi`

Unfortunately, the `--message-format` parameter needs to be passed before the `--`. Like this, it causes the following error:

```
error: failed to run `rustc` to learn about target-specific information
Caused by:
  process didn't exit successfully: `/home/local/sebschmi/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/clippy-driver /home/sebschmi/.cargo/bin/rustc - --crate-name ___ --print=file-names --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg` (exit code: 1)
  --- stderr
  error: Unrecognized option: 'message-format'
Process finished with exit code 101
```

## Steps to reproduce
See problem description.

**Comment by Undin** [comment]
Duplicate #6802
Already fixed in #6805.
The fix will be included in next stable build next week. To try it right now, you can install nightly plugin version. See [instruction](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-quick-start.html#install-nightly) how to do it

**Comment by sebschmi** [comment]
Thank you very much
________________________________
